persist/storer.go

```diff
@@ -56,8 +54,12 @@ type inMemoryStore struct {
 
 // Holder for any entry in the JSON storage
 type jsonEntry struct {
+	entryTimestamp `json:",inline"`
+	Value          interface{}
+}
+
+type entryTimestamp struct {
 	Timestamp int64
-	Value     interface{}
 }
 
```

Comment:
out of curiosity, why this ☝️  change?

 If it's only for the unmarshall in `deleteOldEntries` I wouldn't change this struct but I would create an "anonymous" struct in `deleteOldEntries`

Reply:
> out of curiosity, why this ☝️ change?
Only timestamp is needed for the process of removing old entries. Decoding only this value saves some overhead (depending on the value stored), benchmark was almost half IIRC.

I started with that approach of anonymous and then moved here to have more transparent and hard dependency, but I don't have strong opinion on both, this just felt more safe to me in case of feature change WDYT?

Reply:
Looks good to me , I just changed